docs/src/R_comparison.md

```diff
@@ -1,98 +1,121 @@
+This sections presents examples to help move from R to Julia. Examples show R and Julia code for common operations in survey analysis. <br>
+For the same operation, first the R and then the Julia code is presented. 
 
-In the following examples, we'll compare Julia's performance to R's on the same set of operations.
+## Simple random sample
 
-## Installing and loading the package
-**R**
+The `apisrs` data, which is provided in both `survey` and `Survey`, is used as an example. It's a simple random sample of the Academic Performance Index of Californian schools.
 
-```r
-install.package("survey")
+### 1. Creating a design object
+The following example shows how to construct a design object for a simple random sample. 
+
+```R
 library(survey)
+data(api)
+dsrs = svydesign(id = ~1, data = apisrs, weights = ~pw, fpc = ~fpc)
 ```
 
-**Julia**
 ```julia
-using Pkg
-Pkg.add(url = "https://github.com/xKDR/Survey.jl.git")
 using Survey
+srs = load_data("apisrs")
+dsrs = SimpleRandomSample(srs; popsize = :fpc)
 ```
 
-The following command in the Pkg REPL may also be used to install the package.
+### 2. Mean
+In the following example the mean of the variable `api00` is calculated. 
+
+```R
+svymean(~api00, dsrs)
 ```
-add "https://github.com/xKDR/Survey.jl.git"
+```julia
+mean(:api00, dsrs)
 ```
 
-## API data
+### 3. Total
+In the following example the sum of the variable `api00` is calculated. 
 
-The Academic Performance Index is computed for all California schools based on standardised testing of students. The [data sets](https://cran.r-project.org/web/packages/survey/survey.pdf) contain information for all schools with at least 100 students and for various probability samples of the data. apiclus1 is a cluster sample of school districts, apistrat is a sample stratified by stype.
+```R
+svytotal(~api00, dsrs)
+```
+```julia
+total(:api00, dsrs)
+```
 
-In the following examples, we'll use the apiclus1 data from the api dataset.
+### 4. Quantile
+In the following example the median of the variable `api00` is calculated.
+```R
+svyquantile(~api00, dsrs, 0.5)
+```
+```julia
+quantile(:api00, dsrs, 0.5)
+```
 
-The api dataset can be loaded using the following command:
+### 5. Domain estimation
+In the following example the mean of the variable `api00` is calculated grouped by the variable `cname`. 
 
-**R**
-```r
-data(api)
+```R
+svyby(~api00, ~cname, dsrs, svymean)
 ```
 
-**Julia**
 ```julia
-apiclus1 = load_data("apiclus1")
+by(:api00, :cname, dsrs, mean)
 ```
 
-## svydesign
-[The ```svydesign``` object combines a data frame and all the survey design information needed to analyse it.](https://www.rdocumentation.org/packages/survey/versions/4.1-1/topics/svydesign)
+## Stratified sample
 
-A ```design``` object can be constructed with the following command:
+The `apistrat` data, which is provided in both `survey` and `Survey`, is used as an example. It's a stratified sample of the Academic Performance Index of Californian schools.
```

Comment:
The first phrase is exactly the same as the one for `apisrs`. How about describing the `api` datasets as a whole with this phrase at the beginning of the file, and mentioning there the available datasets? Something like:

"The `api` data, which is provided in both `survey` and `Survey.jl`, is used as an example. It is a collection of 5 datasets:
- `apipop` - entire population
- `apisrs` - simple random sample
- `apistrat` - stratified sample (by `stype`)
- `apiclus1` - cluster sample (school districts)
- `apiclus2` - two-stage cluster sample (schools within districts)
"

I got the descriptions from [here](https://r-survey.r-forge.r-project.org/survey/html/api.html), we could add this as a reference to the `api` data.

Reply:
We can do it in `load_data`

Reply:
I don't think that's good, because `load_data` is a general function, it does not work only with `api`. Since we're doing the example with `api` here, I think it's best to have it here.

Reply:
Okay, in a separate PR, I'll put a section on sample datasets provided with the package. Or put it in the index.